assets/js/components/DeregistrationModal/DeregistrationModal.jsx

```diff
@@ -30,7 +30,12 @@ function DeregistrationModal({
           onClick={onCleanUp}
         >
           <EOS_CLEANING_SERVICES size="base" className="fill-white inline" />
-          <span className="text-white text-sm font-bold pl-1.5">Clean up</span>
+          <span
+            id="cleanup-confirm"
```

Comment:
This is just for testing?
You should maybe use `data-testid` i guess

Reply:
I know the `click` event will bubble up to the `button` element, so it still works, but isn't it more proper to place this ID on the `button`?

In any case, the ID for testing might not even be required, in previous tests we selected the _confirm clean up button_ like this:

```javascript
screen.getAllByRole('button', { name: 'Clean up' })[1]
```

https://github.com/trento-project/web/blob/73f91dca23b50266544b02bced0a3299dfcdfde8/assets/js/components/HostsList.test.jsx#L173

Reply:
@jamie-suse That seems from react's testing library, not sure if I can use it in cypress. In any case, based on the official docs https://docs.cypress.io/guides/references/best-practices, `data-cy` seems to be the preferred way.

Reply:
(or data-test or data-testid)